.github/workflows/windows.yml

```diff
@@ -0,0 +1,26 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Java CI with Gradle
```

Comment:
I don't think that testing windows compiles on every push / PR is needed. Also, it looks like the script uses linux commands anyway, so I don't think it will even work on Windows? This file should probably be removed.

Reply:
it does work anyway ill remove it because java isn't a compiled lang.